packages/aws-sdk-client-mock-jest/package.json

```diff
@@ -25,21 +25,56 @@
     "jest-matchers"
   ],
   "scripts": {
-    "test": "jest --coverage --colors",
+    "test": "pnpm run jest && pnpm run vitest",
+    "jest": "jest --coverage --colors ",
+    "vitest": "vitest run",
     "test-types": "tsd",
     "build:cjs": "tsc -p tsconfig.json",
     "build:es": "tsc -p tsconfig.es.json",
     "prebuild": "rimraf dist/",
     "build": "pnpm run build:cjs && pnpm run build:es",
     "local-publish": "pnpm publish --registry http://localhost:4873/ --no-git-checks"
   },
-  "module": "dist/es/index.js",
-  "main": "dist/cjs/index.js",
-  "types": "dist/types/index.d.ts",
+  "module": "dist/es/jest.js",
+  "main": "dist/cjs/jest.js",
+  "types": "dist/types/jest.d.ts",
+  "exports": {
+    ".": {
+      "require": {
+        "types": "./dist/types/jest.d.ts",
+        "default": "./dist/cjs/jest.js"
+      },
+      "import": {
+        "types": "./dist/types/jest.d.ts",
+        "default": "./dist/es/jest.js"
+      }
+    },
+    "./jest-globals": {
+      "require": {
+        "types": "./dist/types/jestGlobals.d.ts",
+        "default": "./dist/cjs/jestGlobals.js"
+      },
+      "import": {
+        "types": "./dist/types/jestGlobals.d.ts",
+        "default": "./dist/es/jestGlobals.js"
+      }
+    },
+    "./vitest": {
+      "require": {
+        "types": "./dist/types/vitest.d.ts",
+        "default": "./dist/cjs/vitest.js"
+      },
+      "import": {
+        "types": "./dist/types/vitest.d.ts",
+        "default": "./dist/es/vitest.js"
+      }
+    }
+  },
   "files": [
     "dist"
   ],
   "dependencies": {
+    "@vitest/expect": "1.6.0",
```

Comment:
Let's make it `>1.6.0` to not create conflicts with user's or other packages dependencies?

Reply:
Change it to `>1.6.0`